subworkflows/local/vcf_annotate_all/main.nf

```diff
@@ -19,47 +19,44 @@ workflow VCF_ANNOTATE_ALL {
     vep_cache_version
     vep_cache
     vep_extra_files
-    bcftools_annotations
-    bcftools_annotations_index
-    bcftools_header_lines
 
     main:
-    reports = Channel.empty()
-    vcf_ann = Channel.empty()
-    tab_ann = Channel.empty()
-    json_ann = Channel.empty()
-    versions = Channel.empty()
+    def ch_reports = Channel.empty()
+    def vcf_ann = Channel.empty()
+    def tab_ann = Channel.empty()
+    def json_ann = Channel.empty()
+    def versions = Channel.empty()
```

Comment:
do we really need def for channels?

Reply:
Ben said that it's better so I started doing that :)

Reply:
I see